### DISCUSSION TITLE
Playtime: Shortcuts of Reaper be able to work in Playtime as well or addition of 'shortcut key' to open search bar for plugin

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1250] [Ideas]
It would be nice if shortcuts/keys of Reaper would work in Playtime 2 cause currently for example I can click a shortcut button in Reaper mixer or when selected a track in arrangement and it shows me a ‘search bar’ instantly, so I can just search for an instrument or plugin instantly (check this out, I am using this script/feature: https://forum.cockos.com/showthread.php?t=229807 ). It would be great either to be able to have such feature of instant search in Playtime 2 or at least be able to click the same shortcut as in Reaper to have this search bar to pop up cause in Reaper it does work but some how in Playtime 2 it does not…

**Comment by helgoboss** [comment]
In the actions window of REAPER you can make a shortcut **global**. Then it will be usable also within Playtime. Does this solve your issue?

**Reply by joungle123** [reply]
@helgoboss could you guide me a bit? Just cannot find any option to make it global for a shortcut in actions window ...

**Reply by joungle123** [reply]
@helgoboss ok I got the script working globally, finally found it! So the issue persists but it is different now - could you make such that when you select track in Playtime then it is also selected in the mixer in reaper itself? Currently I am able to get the search bar script working but I need to select track in the mixer itself in reaper cause it has no effect and does not work when I select the track in Playtime. The search bar is doing simple thing - allows to quickly load instrument or effect without opening those annoying windows ..

**Reply by joungle123** [reply]
@helgoboss look, I see that it actually selects track in the mixer of Reaper when I select a TOP of the track column name. However it does not select when I click on the column name at the bottom of the Playtime (Playtime mixer). Could you fix this please? I would be very thankful cause majority of the work is actually selecting instruments, add effects and manipulating sound. The recording part is quite easy and requires the not that much time and effort as the selection part, so thats why this is important.

**Reply by helgoboss** [reply]
Will do that in issue #1258.

**Reply by joungle123** [reply]
Thank you!
